cpp/memilio/utils/time_series.h

```diff
@@ -102,6 +102,36 @@ class TimeSeries
         col.tail(expr.rows()) = expr;
     }
 
+    /**
+     * @brief Initialize a TimeSeries with a table.
+     * @param table Consists of a list of time points, each of the form (time, value_0, value_1, ..., value_n) for
+     *     some fixed n >= 0. 
+     */
+    TimeSeries(std::vector<std::vector<FP>> table)
```

Comment:
I used vector<vector> since it allows using bracket initializers, which do not work with Eigen::Matrix AFAIK. The main use case for that is the abm_braunschweig.cpp, which sets a lot of these.

Reply:
That makes sense, and I was annoyed by this a few times already. Maybe we should have a factory function for matrices that take vector<vector<>> or similar.

Reply:
Do you already have something in mind, or should we open another issue?

Reply:
A simple version would be like the constructor here, just in it's own function. But I guess this could use a bit more thought about the API, so it's generic but still easy to use.